packages/relay-runtime/store/__tests__/RelayModernEnvironment-ExecuteMutationWithMatch-test.js

```diff
@@ -652,9 +652,9 @@ describe.each(['RelayModernEnvironment', 'MultiActorEnvironment'])(
         it('optimistically creates @match fields and loads resources', () => {
           operationLoader.load.mockImplementationOnce(() => {
             return new Promise(resolve => {
-              setImmediate(() => {
+              setTimeout(() => {
                 resolve(markdownRendererNormalizationFragment);
-              });
+              }, 0);
```

Comment:
Only use of `setImmediate` in the tests. No test failures from this change - possibly there are console messages related to it but they were truncated in the test output.